Services/UserService/Startup.cs

```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHttpsRedirection();
 
             app.UseRouting();
-            app.UseMiddleware<TokenMiddleware>();
+            //app.UseMiddleware<TokenMiddleware>();
```

Comment:
I think that here it is necessary to add TODO because antoher person can forget about this comment.

Reply:
> I think that here it is necessary to add TODO because antoher person can forget about this comment.

maybe

Reply:
> I think that here it is necessary to add TODO because antoher person can forget about this comment.

I did